README.md

```diff
@@ -49,6 +49,7 @@ jobs:
           version-prefix: "v"  # useful for repos that terraform modules where the versions are like "v0.2.4".
           settings-file: cicd/settings.json
           base-commit-sha: ${{ github.event.pull_request.base.sha || github.event.before }} # if project not use squash commits
+          always-increase-version: "false" # always increase version even if there are no application changes (i.e. documentation changes, build changed etc will also increase version), default is true
```

Comment:
@MartinMitro , let's rename the input to `increase-version-if-release-none`  with a default value of true. In that way we link this to the release type in the config.json file:
```json
    "docs": {
        "release": "none"
    }
```